spectator/registry.cc

```diff
@@ -4,11 +4,15 @@
 
 namespace spectator {
 
-Registry::Registry(Config config) noexcept
```

Comment:
Is this constructor considered a public api that we would need to worry about changing?